cmd/issuer-rest/startcmd/start.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package startcmd
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/spf13/cobra"
+
+	"github.com/trustbloc/edge-service/pkg/restapi/issuer/operation"
+	cmdutils "github.com/trustbloc/edge-service/pkg/utils/cmd"
+)
+
+const (
+	hostURLFlagName      = "host-url"
+	hostURLFlagShorthand = "u"
+	hostURLFlagUsage     = "URL to run the issuer-rest instance on. Format: HostName:Port."
+	hostURLEnvKey        = "ISSUER_REST_HOST_URL"
+)
+
+var errMissingHostURL = errors.New("host URL not provided")
+
+type edgeStoreParameters struct {
```

Comment:
Change name to issuerRestParameters or something similar

Reply:
done